storage_service/templates/snippets/pipelines_table.html

```diff
@@ -14,7 +16,9 @@
         <td><a href="{% url 'locations:pipeline_detail' pipeline.uuid %}">{{ pipeline.uuid }}</a></td>
         <td>{{ pipeline.description }}</td>
         <td>{{ pipeline.enabled|yesno:_("Enabled,Disabled") }}</td>
-        <td><a href="{% url 'locations:pipeline_edit' pipeline.uuid %}">{% trans "Edit" %}</a> | <a href="{% url 'locations:pipeline_switch_enabled' pipeline.uuid %}?next={{ request.path }}">{{ pipeline.enabled|yesno:_("Disable,Enable") }}</a> | <a href="{% url 'locations:pipeline_delete' pipeline.uuid %}?next={{ request.path }}">{% trans "Delete" %}</a></td>
+        {% if perms.locations.change_pipeline %}
```

Comment:
Should the `Delete` link contained here be protected with the `locations.delete_pipeline` permission?